docs/rest.md

```diff
@@ -661,3 +661,25 @@ Sample input:
     “slot”: {“snap”: “keyboard-lights”, “slot”: “capslock-led”}
 }
 ```
+
+## /2.0/events
+
+### GET
+
+* Description: Websocket upgrade
```

Comment:
The description is a bit vague IMHO. Websocket upgrade is the mechanism that happens there but surely that's not what this method is doing.